controller/handler-test/compliance-values.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const test = require('ava');
+const sinon = require('sinon');
+const proxyquire = require('proxyquire');
+const QueryTypes = {
+    SELECT: 'select'
+};
+
+test.cb('when patients exist in trial', (t) => {
+    const model = sinon.stub();
+    const query = sinon.stub();
+
+    model
+    .withArgs('trial')
+    .returns({
+        findById () {
+            return Promise.resolve({
+                name: 'example',
+                description: 'example',
+                IRBID: 1,
+                IRBStart: new Date(),
+                IRBEnd: new Date(),
+                targetCount: 1,
+                patientPinCounter: 1
+            });
+        }
+    });
+
+    model
+    .withArgs('stage')
+    .returns({
+        findAll () {
+            return Promise.resolve([]);
+        }
+    });
+
+    query
+    .onFirstCall()
```

Comment:
There is only one query call so this does not need first, second and, third stubs.